srcpkgs/haxor-news/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'haxor-news'
+pkgname=haxor-news
+version=0.4.2
+revision=1
+noarch=yes
+build_style=python3-module
+pycompile_module="haxor_news"
+hostmakedepends="python3-setuptools"
+depends="python python3-click python3-colorama python3-requests python3-Pygments python3-prompt_toolkit python3-six"
+short_desc="Browse Hacker News like a haxor: A Hacker News command line interface"
+maintainer="Crestwave <crest.wave@yahoo.com>"
+license="Apache-2.0"
+homepage="https://github.com/donnemartin/haxor-news"
+distfiles="${homepage}/archive/${version}.tar.gz"
+checksum="fa35cea905c34b6975527b2e6e9d6b020a6cad6c664c8483128e10a07d998314"
```

Comment:
no need to be quoted

Reply:
Ah, the examples in the manual had them quoted so I thought that they were preferred. Thanks for the review!